src/data-table/DataTableBody.tsx

```diff
@@ -158,6 +171,7 @@ export function DataTableBody<T extends ReferenceObject>(props: DataTableBodyPro
                     onContextMenu={event => handleClick(event)}
                     role="checkbox"
                     selected={checked || indeterminate}
+                    style={style}
```

Comment:
I am unable to make this work, I pass a style from the app, but it's not rendered on the `tr` tag here.

Reply:
Not sure, I am able to do it on ```metadata-sync`` and I did not require the ``!important``, Perhaps ``project-monitoring`` has something in the global stylesheet?

Reply:
It was me, I had to remove the "!important", cannot be used in an inline css. Working fine!